### DISCUSSION TITLE
日志建议

### DISCUSSION THREAD

**Discussion by Clorasun** [DUpdateSystem/UpgradeAll#235] [General]
crash不能保留日志，检测应用和偶尔进入应用直接卡死

**Comment by xz-dev** [comment]
已加入开发计划
https://github.com/DUpdateSystem/UpgradeAll/issues/236
https://github.com/DUpdateSystem/UpgradeAll/issues/237

**Comment by xz-dev** [comment]
如果发现卡死问题或其他问题，可能是开了过多的应用市场模式的问题
解决方法是：先开飞行模式，然后在软件源界面关闭部分源的应用市场模式

If you find any problem, it may be the problem of enabling too many application market mode.
The solution is:
1. open the flight mode
2. turn off the application market mode of some sources in the software source interface
